packages/website/src/bin/page-generator/index.js

```diff
@@ -369,15 +377,18 @@ module.exports = async function generatePages(
 
   const pkgOpts = {};
 
-  if (typeof options.useManifests !== 'undefined') {
+  if (options.useManifests) {
     pkgOpts.useManifests = options.useManifests;
   }
-  if (typeof options.showSubExamples !== 'undefined') {
+  if (options.showSubExamples) {
     pkgOpts.showSubExamples = options.showSubExamples;
   }
-  if (typeof options.showExamples !== 'undefined') {
+  if (options.showExamples) {
```

Comment:
I think we need the `undefined` checks here so that `pkgOpts` are always set. Otherwise, false values won't be set which will cause `defaultOptions` that are `true` in `getPackageInfo` to be used. One such example is `showExamples` which is true by default, setting it to `false` in config won't disable that option unless we use undefined checks (or just `!= null`)

Reply:
I've updated these to use a loose `null` check, and added tests for them too

Reply:
Yeah, explicitly checking for not undefined is important, rather than just truthiness. It would be good to add a comment about this so we don't actually erase this in the future.